web/massastation/src/i18n/en_US.json

```
@@ -11,7 +11,7 @@
       "banner": "I want to create and install my own module!",
       "description": "How to do it: {url}",
       "title": " Manual installation",
-      "subtitle": "Copy the link of the module zip",
+      "subtitle": "Paste the link of the module zip",
```

you added a comment in figma?

```suggestion
      "subtitle": "Paste the link of the zip module",
```